doc/sphinx/source/frameworks/tensorflow_connect.rst

```diff
@@ -1,14 +1,134 @@
-.. _ngraph_bridge repo: https://github.com/tensorflow/ngraph-bridge  
-.. _DSO: http://csweb.cs.wfu.edu/%7Etorgerse/Kokua/More_SGI/007-2360-010/sgi_html/ch03.html
+If you are already familiar with TensorFlow\*, the easiest way to get started 
+is to use the latest `prebuilt nGraph-TensorFlow bridge`_. You can install 
+TensorFlow and nGraph to a virtual environment; otherwise, the code will install 
+to a system location.
+
+.. important:: The latest version of TensorFlow may be greater than the version 
+   named below.
+
+.. code-block:: console
+   
+   pip install -U tensorflow==1.14.1
+   pip install -U ngraph-tensorflow-bridge
+
+That's it!  Now you can test the installation by running the following command:
+
+.. code-block:: python
+
+   python -c "import tensorflow as tf; print('TensorFlow version: ',tf.__version__);
+   import ngraph_bridge; 
+   print(ngraph_bridge.__version__)"
+
+Detailed examples are located in the `ngraph_bridge examples`_ directory. 
+
+
+Build your own nGraph-TensorFlow bridge 
+---------------------------------------
+
+A slightly more involved option is to build the latest nGraph with a binary 
+TensorFlow; use this option if you know you need features newer than those 
+available in the prebuilt binary. Review the latest :doc:`../project/release-notes` 
+for the most recent changes. 
+
+To build your own bridge, follow these steps:
+
+#. **Clone the nGraph-TensorFlow bridge repo**
+
+    .. code-block:: console
+
+       git clone https://github.com/tensorflow/ngraph-bridge.git
+       cd ngraph-bridge
+
+#. The script we need to run assumes you're running the ``pip`` implementation 
+   of ``virtualenv``, on Python3.5 or greater.  The quickest way to get this on 
+   your system is:
+
+   .. code-block:: console
+
+       python3 -m pip install virtualenv==16.0.0 --user
+
+
+#. Run the script 
+
+   .. code-block:: console
+
+      python3 build_ngtf.py --use_prebuilt_tensorflow
+
+#. Now test the build by executing the following commands:
+
+   .. code-block:: console
+
+       source build_cmake/venv-tf-py3/bin/activate
+       PYTHONPATH=`pwd` python3 test/ci/buildkite/test_runner.py \
+            --backend CPU \
+            --artifacts ./build_cmake/artifacts/ \
+            --test_resnet
+
+That's it! Now you can take a look at and start experimenting with the detailed 
+located in the `ngraph_bridge examples`_ directory. 
+
+
+Building nGraph bridge from source
+----------------------------------
+
+The other way to build from source is to run without prebuilt options; try this if running 
+the script above with the ``--use_prebuilt_tensorflow`` option doesn't work.
+
+#. **Clone the nGraph-TensorFlow bridge repo**
+
+   .. code-block:: console
+
+      git clone https://github.com/tensorflow/ngraph-bridge.git
+      cd ngraph-bridge
+      git checkout v0.14.0
+      cd ../
+
+#. Install Bazel v ``0.24.1``; Bazel is a TensorFlow dependency:
```

Comment:
For `v0.14.0` bazel version is `0.21.0`

Reply:
Also `0.14.1` is the latest.

Reply:
There was some conversation in the slack channel earlier, I'm hopeful we got this right finally.